pkgs/development/tools/cloudfoundry-cli/default.nix

```diff
@@ -35,7 +42,7 @@ buildGoPackage rec {
 
   installPhase = ''
     install -Dm555 out/cf "$out/bin/cf"
-    installShellCompletion --bash "$src/ci/installers/completion/cf"
+    installShellCompletion --bash $bashCompletionScript
```

Comment:
This is installing the completion as `cf7`, not sure if it needs to be renamed?

Reply:
It seems to work for me as cf7, and that's what the official deb packages built by pivotal seem to put it under.